Header header
float64 rel_p_x
float64 rel_p_y
float64 rel_p_z
float64 rel_v_x
float64 rel_v_y
float64 rel_v_z
float64 rel_yaw
float64 rel_a_x
float64 rel_a_y
float64 rel_a_z
float64 roll
float64 pitch
float64 yaw
float64 v_z
float64 roll_rate
float64 pitch_rate
float64 yaw_rate




